include/PR/sched.h

```diff
@@ -0,0 +1,74 @@
+#ifndef SCHED_H
```

Comment:
The scheduler used in Zelda 64 is different to the scheduler of libultra. I don't think this should be in the PR directory.